backend/src/test/java/com/woowacourse/gongcheck/acceptance/SpaceAcceptanceTest.java

```diff
@@ -82,7 +82,22 @@ class SpaceAcceptanceTest extends AcceptanceTest {
     }
 
     @Test
-    void 공간을_삭제한다() {
+    void 단일_Space를_조회한다() {
+        GuestEnterRequest guestEnterRequest = new GuestEnterRequest("1234");
+        String token = 토큰을_요청한다(guestEnterRequest);
+
```

Comment:
```suggestion
    void 단일_Space를_조회한다() {
        // Host 인가 구현 전까지 임시로 사용
        GuestEnterRequest guestEnterRequest = new GuestEnterRequest("1234");
        String token = 토큰을_요청한다(guestEnterRequest);

```
원래 여기서 쓰는 인증 방식이 아닌데, 임시로 쓰고 있으니 주석으로 특이사항을 표현해주면 좋을 것 같습니다.